templates/registration/login.html

```diff
@@ -39,6 +40,14 @@
                         </div>
                         <!-- /.card-footer -->
                     </form>
+                    {% endif %}
+                    {% if openid_active %}
+                    <div class="card-footer">
+                        <div class="text-center">
+                            <a class="btn" style="background-color: #6ad3c2" href="{% url "social:begin" "oidc" %}" role="button"><strong>{{ openid_btn_text }}</strong></a>
```

Comment:
Could we use a pre [defined color](https://adminlte.io/docs/3.0/layout.html) of adminlte framework?

Reply:
Just to check, since this would make it inconsistent with the password form sign-in button. Do you want that one adjusted as well?

Reply:
Oh sorry I haven't seen  that it was like that for the first buton.
You can leave it like this I guess then.